### ISSUE TITLE
vscode-eslint is not recognizing eslint v2.x new module flag

### ISSUE THREAD

**Issue by Alexisvt** [Microsoft/vscode-eslint#73]
Hi I have on my machine eslint v 2.8 and currently I'm working with a js project that have es2015 syntax!.
When I lint my code with the eslint terminal tool everything is correct but when I see the same code in visual studio code using the eslint plugin is showing an error on a file that have `import` statement.
Currently my `.eslintrc.json` file see like this:

```json
{
  "env": {
    "browser": true,
    "node": false,
    "es6": true
  },
  "parserOptions": {
    "ecmaVersion": 6,
    "sourceType": "module",
    "ecmaFeatures": {
    }
  }
}
```
and my code looks like this:
```js
import "ngDraggable";
import angular from "angular";
import "./klondike/game.css!";

import "klondike/klondike.js";

angular.module("solitaire", ["klondike", "ngDraggable"]);

angular.element(document).ready( () => {
    angular.bootstrap(document, ["solitaire"]);
});
```
Now when I change the rules to eslint v 1.x to allow the modules vs code now doesn't show any error on the import statement anymore. But this is not desirable by obvious reasons. 
```json
{
  "env": {
    "browser": true,
    "node": false,
    "es6": true
  },
  "ecmaFeatures": {
    "modules": true
  }
}
```
Do you know If I'm missing something or perhaps is this an error? Thanks!

By the way if you want to read more about this rule you can go to this link: [migrating-to-2.0.0#language-options](http://eslint.org/docs/user-guide/migrating-to-2.0.0#language-options)

**Comment by dbaeumer** [comment]
@Alexisvt sorry for the late answer but I was out on vacation. 

This is not desirable. The eslint extension looks for the eslint library at the following locations:

- in the node_modules folder of the workspace folder opened in vscode
- globally installed.

Could it be that when run in the terminal you use a globally installed eslint and you have a local installed eslint as well. That would explain the different behavior.

**Comment by Alexisvt** [comment]
Hi @dbaeumer yes! That was the reason, my bad!! I checked the **package.json** file and I found a devDependencies to a eslint **v^1.5.1**. Thanks for the help :smile:
